apps/openmw/CMakeLists.txt

```diff
@@ -140,18 +140,50 @@ target_link_libraries(openmw
 )
 
 if (ANDROID)
+    set (OSG_PLUGINS
+        -Wl,--whole-archive
+        ${OSG_PLUGINS_DIR}/libosgdb_dds.a
+        ${OSG_PLUGINS_DIR}/libosgdb_bmp.a
+        ${OSG_PLUGINS_DIR}/libosgdb_tga.a
+        ${OSG_PLUGINS_DIR}/libosgdb_gif.a
+        ${OSG_PLUGINS_DIR}/libosgdb_jpeg.a
+        ${OSG_PLUGINS_DIR}/libosgdb_png.a
+        -Wl,--no-whole-archive
+    )
     target_link_libraries(openmw
         EGL
         android
         log
         dl
         MyGUIEngineStatic
-        cpufeatures
         BulletCollision
         LinearMath
+        z
+        osg
+        osgDB
+        osgAnimation
+        osgText
+        osgUtil
+        osgShadow
+        ${OPENSCENEGRAPH_LIBRARIES}
+        ${OSG_PLUGINS}
+        ${Boost_SYSTEM_LIBRARY}
+        ${Boost_THREAD_LIBRARY}
+        ${Boost_FILESYSTEM_LIBRARY}
+        ${Boost_PROGRAM_OPTIONS_LIBRARY}
+        jpeg
+        gif
+	png
     )
 endif (ANDROID)
 
+
+if (OPENGL_ES)
+    target_link_libraries(openmw
+        ${OPENGLES_gl_LIBRARY}
+    )
+endif (OPENGL_ES)
```

Comment:
This line shouldn't be needed if you add the OPENGLES_gl_LIBRARY to components/CMakeLists.txt as suggested in my last comment.